spec/features/small_feature_spec.rb

```diff
@@ -0,0 +1,53 @@
+RSpec.describe "small_features", type: :feature do
```

Comment:
This should maybe be called something more descriptive or the tests could be put in more appropriate files.